### PR TITLE
CLN: noqa removal

### PR DESCRIPTION
**Pull request by kaathi524**
xref #29216

### CODE REVIEW

**Comment by jbrockmendel** [comment]
LGTM, cc @WillAyd

**Review by jreback — APPROVED** [review]

File: pandas/tests/dtypes/test_inference.py [hunk 1]
```diff
@@ -62,46 +62,46 @@ def coerce(request):
 # collect all objects to be tested for list-like-ness; use tuples of objects,
 # whether they are list-like or not (special casing for sets), and their ID
 ll_params = [
-    ([1], True, "list"),  # noqa: E241
-    ([], True, "list-empty"),  # noqa: E241
-    ((1,), True, "tuple"),  # noqa: E241
```
Comment (jorisvandenbossche) [inline pandas/tests/dtypes/test_inference.py]: @kaathi524 what's the reason this is not necessary any more?

Comment (kaathi524) [reply pandas/tests/dtypes/test_inference.py]: @jorisvandenbossche Yeah. According to Issue #29607 , Need to remove those noqa.

**Review by jorisvandenbossche — APPROVED** [review]

File: pandas/tests/dtypes/test_inference.py (hunk 1 above)
Comment (jorisvandenbossche) [reply pandas/tests/dtypes/test_inference.py]: OK, thanks!

**Comment by jorisvandenbossche** [comment]
@kaathi524 thanks for the PR!

**Pull request merged**